play/src/front/Components/RefreshPrompt.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+    import { onMount } from "svelte";
+    import { mapUploadRefreshNeededCommentStore } from "../Stores/MapEditorStore";
+
+    let timeToRefreshSeconds = 30;
```

Comment:
Any chance we can make this timer configurable? (sent from the back/pusher along the message?)